command/cmd_copy.go

```diff
@@ -19,6 +19,38 @@ import (
 	"github.com/peak/s5cmd/storage"
 )
 
+var copyHelpTemplate = `Name:
+	{{.HelpName}} - {{.Usage}}
+
+Usage:
+	{{.HelpName}} [options] source destination
+
+Options:
+	{{range .VisibleFlags}}{{.}}
+	{{end}}
+Examples:
+	1. Download an S3 object to working directory
+		 > s5cmd {{.HelpName}} s3://bucket/prefix/object.gz .
+
+	2. Download an S3 object and rename
+		 > s5cmd {{.HelpName}} s3://bucket/prefix/object.gz myobject.gz
+
+	3. Download all S3 objects to a directory
+		 > s5cmd {{.HelpName}} s3://bucket/* target-directory/
+
+	4. Upload a file to S3 bucket
```

Comment:
can we add one more example for the wildcard usage on s3 upload?

Reply:
done.